client/app/services/auth.js

```diff
@@ -24,6 +24,7 @@ function getLocalSessionData() {
 
 function AuthService($window, $location, $q, $http) {
   const Auth = {
+    logger,
```

Comment:
I'm not sure it's worth passing the logger around. It can be as easily created where needed, no?